### PR TITLE
refactor: replace svg to SfIcon in SfRating component

### PR DESCRIPTION
**Pull request by martaradziszewska**
# Related issue
Closes: #621

# Scope of work
Add star icon to icons.js
Implemented SfIcon in SfRating replacing svg

# Screenshots of visual changes

# Checklist

- [x] I followed [composition rules](https://docs.storefrontui.io/contributing/coding-guidelines.html#component-rules) for my component
- [x] I tested the component in most common device sizes (can be tested in Storybook [from viewport addon in top menu](https://github.com/storybooks/storybook/tree/master/addons/viewport))

### CODE REVIEW

File: packages/vue/src/components/atoms/SfRating/SfRating.html [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <div class="sf-rating">
   <!--@slot custom icon for finalScore. Provide single icon that will be automatically repeated -->
   <slot name="icon-positive" v-if="finalScore && finalMax" v-for="index in parseInt(finalScore, 10)">
-    <svg width="14" height="14" viewBox="0 0 11 10" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M5.5 0L6.73483 3.80041H10.7308L7.49799 6.14919L8.73282 9.94959L5.5 7.60081L2.26718 9.94959L3.50201 6.14919L0.269189 3.80041H4.26517L5.5 0Z" class="sf-rating__icon-positive"/>
+    <SfIcon size="14px" icon="star" color="green-primary"></SfIcon>
```
Comment (filrak) [inline packages/vue/src/components/atoms/SfRating/SfRating.html]: What if someone wants to change the color of active icons? color should be set to default accent imho

Comment (martaradziszewska) [reply packages/vue/src/components/atoms/SfRating/SfRating.html]: It is possible to put color as a prop on SfIcon not only with our standard colors, or some custom color such as `#fff`, `rgb(255,255,255)`), and `lightgray` or nothing. What do you think ?

Comment (martaradziszewska) [reply packages/vue/src/components/atoms/SfRating/SfRating.html]: on 2nd thoughts it is better to leave default colors as you said :) I restored styles for it, thanks!

File: packages/vue/src/components/atoms/SfRating/SfRating.html [hunk 2]
```diff
@@ -1,12 +1,12 @@
 <div class="sf-rating">
   <!--@slot custom icon for finalScore. Provide single icon that will be automatically repeated -->
   <slot name="icon-positive" v-if="finalScore && finalMax" v-for="index in parseInt(finalScore, 10)">
-    <svg width="14" height="14" viewBox="0 0 11 10" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M5.5 0L6.73483 3.80041H10.7308L7.49799 6.14919L8.73282 9.94959L5.5 7.60081L2.26718 9.94959L3.50201 6.14919L0.269189 3.80041H4.26517L5.5 0Z" class="sf-rating__icon-positive"/>
+    <SfIcon size="14px" class="sf-rating__icon-positive" icon="star"></SfIcon>
```
Comment (pspaczek) [inline packages/vue/src/components/atoms/SfRating/SfRating.html]:
```suggestion
    <SfIcon size="14px" class="sf-rating__icon-positive" icon="star"/>
```

File: packages/vue/src/components/atoms/SfRating/SfRating.html [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
     </svg>
   </slot>
   <!--@slot custom icon for negative (left) finalScore. Provide single icon that will be automatically repeated -->
   <slot name="icon-negative" v-if="finalScore && finalMax" v-for="index in (parseInt(finalMax, 10)-parseInt(finalScore, 10))">
-    <svg width="14" height="14" viewBox="0 0 11 10" fill="none" xmlns="http://www.w3.org/2000/svg"><path d="M5.5 0L6.73483 3.80041H10.7308L7.49799 6.14919L8.73282 9.94959L5.5 7.60081L2.26718 9.94959L3.50201 6.14919L0.269189 3.80041H4.26517L5.5 0Z" class="sf-rating__icon-negative"/>
+    <SfIcon size="14px" class="sf-rating__icon-negative"  icon="star"></SfIcon>
```
Comment (pspaczek) [inline packages/vue/src/components/atoms/SfRating/SfRating.html]:
```suggestion
    <SfIcon size="14px" class="sf-rating__icon-negative"  icon="star/>
```

Comment (martaradziszewska) [reply packages/vue/src/components/atoms/SfRating/SfRating.html]: ok thanks!

**Comment by pspaczek** [comment]
@martaradziszewska  I think about one improvement. SfIcon uses `--icon-color` what did you think about overriding it instead of use fill? Probably you should use it like that
``` scss
&__icon-negative{
  &.sf-icon{
    --icon-color: ...
  }
}
```
When we use fill we do something simmilar but not realy :)

**Pull request merged**